src/py42/sdk/queries/fileevents/filters/cloud_filter.py

```diff
@@ -3,23 +3,49 @@
 
 
 class Actor(FileEventFilterStringField):
+    """Class that filters events by the cloud service username of the event originator
+    (applies to cloud data source events only).
+    """
+
     _term = u"actor"
 
 
 class DirectoryID(FileEventFilterStringField):
+    """Class that filters events by unique identifier of the cloud drive or folder where the event
+     occurred (applies to cloud data source events only).
+    """
+
     _term = u"directoryId"
 
 
 class Shared(QueryFilterBooleanField):
+    """Class that filters events by the shared status of the file at the time the event occurred
+    (applies to cloud data source events only).
+    """
+
     _term = u"shared"
 
 
 class SharedWith(FileEventFilterStringField):
+    """Class that filters events by the users files were explicitly shared with at the time of the
+    event (applies to cloud data source events only).
+    """
+
     _term = u"sharedWith"
 
 
 class SharingTypeAdded(FileEventFilterStringField):
+    """Class that filters results to include events where a file's sharing permissions were
+    increased to increase exposure (applies to cloud data source events only).
```

Comment:
The word `increased` so close to `increase` is a little weird.

My suggestion: `... were increased, causing a greater risk of exposure...`

Reply:
Updated to use similar wording as our support docs, much clearer.